easybuild/easyconfigs/z/zarr/zarr-2.1.4-foss-2017a-Python-2.7.13.eb

```diff
@@ -0,0 +1,47 @@
+easyblock = "PythonPackage"
+
+name = 'zarr'
+version = '2.1.4'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://zarr.readthedocs.io/en/stable/'
+description = """Zarr is a Python package providing an implementation of compressed, chunked, N-dimensional arrays, 
+ designed for use in parallel computing."""
+
+toolchain = {'name': 'foss', 'version': '2017a'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
```

Comment:
Please add `SHA256` checksum

Reply:
Are the extensions dependencies for `zarr`? If so, include `zarr` as last extension (and use the `Bundle` easyblock), rather than listing sources here (since then it will pull in dependencies automatically first before starting to install extensions).